src/NoteCommandFactory.js

```diff
@@ -0,0 +1,66 @@
+var emitter = require('./utils/emitter');
```

Comment:
The `NoteCommandFactory.js` and `NoteController.js` filenames are in camel case while all other names are separated by dashes. What convention should we use?